docker/README.md

```diff
@@ -120,199 +78,414 @@ mv license.key "${LOCAL_LICENSE_FILE_DIR}/license"
 > picked up immediately, you may need to restart the `teams-cas` and
 > `teams-api` services.
 
-We publish the following FiftyOne Enterprise private images to Docker Hub:
+## Step 2: Choose Authentication Mode
+
+FiftyOne Enterprise offers two authentication modes:
+
+- `legacy-auth` → Choose this mode if using Auth0 for user authentication and authorization.
+- `internal-auth` → Choose this mode when in an airgapped deployment - aigapped deployments 
+will not require network egress to external services.
+
+### 👉 Choose your mode
+
+Navigate into the appropriate directory:
+
+```bash
+cd legacy-auth       # or internal-auth
+```
+
+## Step 3: Configure Environment
+
+### 1. Copy the template `.env` file:
+
+```
+cp env.template .env
+```
+
+### 2. Fill out required values in `.env`
+
+At minimum, configure:
+
+- `BASE_URL` / `AUTH0_BASE_URL` - Your WebApp URL
+- `FIFTYONE_API_URI` - Your API URL (can be the same as webApp URL if using path-based routing)
+- `FIFTYONE_DATABASE_URI` – Your MongoDB connection URI
+- `FIFTYONE_ENCRYPTION_KEY` – Used to encrypt storage credentials
+
+> 🔑 To generate a key:
+
+```
+from cryptography.fernet import Fernet
+print(Fernet.generate_key().decode())
+```
+
+- `LOCAL_LICENSE_FILE_DIR` – Path where the license is mounted
```

Comment:
The continuation of the list feels disjointed. Instead, before the list, we could include instructions for how to create the key. And then keep the list (or table) continuous.